googleearth: adding application launcher and icons

fusion809 opened the pull request:
I am adding an application launcher (desktop configuration file) and icons to the appropriate locations in the `share/` directory for the `googleearth` package. I am merely using those already in `opt/`, with the only modification being made to them being to make the `Exec=` line in the desktop configuration file point to the correct executable for the program.

###### Motivation for this change
Presently the `googleearth` package provides no application launcher, so the only way to start it is by manually executing `googleearth` as a command. In this PR I am rectifying this.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

fusion809 commented:
Better CC the maintainer, @markus1189.

markus1189 commented:
Sounds good! Can't test this because I don't have a desktop environment ;)

fusion809 commented:
What about with Rofi? Rofi will show you it.

markus1189 commented:
But rofi does only scan PATH and show me the regular `googleearth` executable unless there is a mode I don't know

fusion809 commented:
Well, for me I use:

```bash
rofi -show drun -drun-icon-theme "Yosa Max" -show-icons
```

under KDE and i3 and it shows me:

![screenshot_20190124_220915](https://user-images.githubusercontent.com/4717341/51677274-cce54600-2024-11e9-9cad-efd42d0844d1.png)

Of course, you will need an installed icon theme to replace "Yosa Max" with.

markus1189 commented:
Nice, didn't now about the `show-icons` flag.  Seems to work fine!

fusion809 committed:
googleearth: adding application launcher and icons
I am adding an application launcher, and icons for the googleearth
package. They are essentially just those in opt/ with the only
adjustment being to make the launcher execute the correct executable
file.

fusion809 commented:
Just pushed a commit to include an icon in `$out/share/pixmaps` too, just in case for some reason the icons in hicolor are ignored.

worldofpeace commented:
Thanks @fusion809 :sparkles:
